data/tutorials/getting-started/gs_01_a_first_hour_with_ocaml.md

```diff
@@ -4,8 +4,7 @@ title: Your First Day with OCaml
 description: >
   Discover the OCaml programming language in this longer tutorial that takes you
   from absolute beginner to someone who is able to write programs in OCaml.
-category: "getting-started"
-date: 2021-08-06T17:11:00-00:00
+category: "Getting Started"
```

Comment:
I think it's better to keep IDs here and transform that to the actual text in the frontend.

Reply:
Then we need to keep a map from ids to names somewhere in the frontend. I chose this way because it's more flexible and there's less places to update simultaneously when making changes. This is just the grouping of items in the sidebar.

Though, if it's important, I can change this.